core/src/main/java/org/elasticsearch/ingest/IngestModule.java

```diff
@@ -81,4 +83,10 @@ public void registerProcessor(String type, BiFunction<Environment, TemplateServi
     public static boolean isIngestEnabled(Settings settings) {
         return settings.getAsBoolean("node.ingest", true);
     }
+
+    public static boolean isIngestEnabled(ImmutableOpenMap<String, String> nodeAttributes) {
```

Comment:
it's here because I want it to have the same exact logic of the other method that's already here. I am afraid they would go out of sync, I will add unit tests for that though.